src/rust/engine/src/scheduler.rs

```diff
@@ -163,7 +163,7 @@ impl Scheduler {
       let mut entry = sizes.entry(k.workunit_name()).or_insert_with(|| (0, 0));
       entry.0 += 1;
       entry.1 += {
-        std::mem::size_of_val(&k)
+        std::mem::size_of_val(k)
```

Comment:
This change is due to a Clippy error: 

```
error: argument to `std::mem::size_of_val()` is a reference to a reference
   --> src/scheduler.rs:166:9
    |
166 |         std::mem::size_of_val(&k)
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: dereference the argument to `std::mem::size_of_val()` to get the size of the value instead of the size of the reference-type
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#size_of_ref
note: the lint level is defined here
   --> src/lib.rs:7:3
    |
7   |   clippy::all,
    |   ^^^^^^^^^^^
    = note: `#[deny(clippy::size_of_ref)]` implied by `#[deny(clippy::all)]`
```